src/Php72/README.md

```diff
@@ -0,0 +1,17 @@
+Symfony Polyfill / Php72
+========================
+
+This component provides functions unavailable in releases prior to PHP 7.2:
+
+- [`stream_isatty`](https://php.net/stream_isatty)
+- [`sapi_windows_vt100_support`](https://php.net/sapi_windows_vt100_support)
+- [`utf8_encode`](https://php.net/utf8_encode)
+- [`utf8_decode`](https://php.net/utf8_decode)
```

Comment:
Maybe just add a small note saying `(available in the optional XML extension earlier)` ?